rb_ws/src/buggy/scripts/2d_sim/controller_2d.py

```diff
@@ -8,6 +9,7 @@ class Controller:
     def __init__(self, buggy_name):
         self.steering_publisher = rospy.Publisher(buggy_name + "/input/steering", Float64, queue_size=10)
         self.velocity_publisher = rospy.Publisher(buggy_name + "/velocity", Float64, queue_size=10)
+        self.manual_velocity_publisher = rospy.Publisher(buggy_name + "/man_velocity", Float64, queue_size=10)
```

Comment:
the controller file is meant to expose a rostopic for controlling the buggy. Ideally we should be able to stick with the same topic instead of publishing a separate topic for manual velocity. 

If we are only going to use one velocity source at a time, why can't manual velocity use the "/velocity" topic? This would also save you from adding manual control flag to the engine.py and doing the check in update_velocity. -> ideally engine.py listens on one steering topic and one velocity, and it executes the commands on that topic without caring about who publishes it.

I understand that you added this because you are concerned about topic collision with the autogenerated velocity, but we can always make sure we are not running both nodes at the same time: just configure the launch files to do so.  (don't worry about launch files in this PR, just don't add the deprecated sim_2d back and it's all good)